homebrew_formula/ballerina.rb

```diff
@@ -0,0 +1,20 @@
+class Ballerina < Formula
```

Comment:
@naaa760 
We already have this in [homebrew source side ](https://github.com/Homebrew/homebrew-core/blob/master/Formula/b/ballerina.rb)
Do you have any reason to keep it here?